bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -164,21 +164,22 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "true"
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
-    olm.skipRange: '>=0.0.0 <99.0.0'
+    olm.skipRange: '>=0.0.0 <1.2.2'
     operatorframework.io/suggested-namespace: openshift-adp
     operators.openshift.io/infrastructure-features: '["Disconnected"]'
     operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
       Platform Plus"]'
     operators.operatorframework.io/builder: operator-sdk-v1.23.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    replaces: oadp-operator.v1.2.1
```

Comment:
Just noticed this is annotations.. will push another one to move it to [.spec.replaces](https://sdk.operatorframework.io/docs/olm-integration/generation/#csv-fields:~:text=spec.replaces%20(user)%3A%20the%20name%20of%20the%20CSV%20being%20replaced%20by%20this%20CSV.)